openedx/core/djangoapps/content_libraries/collections/rest_api/v1/tests/test_views.py

```diff
@@ -0,0 +1,256 @@
+"""
+Tests Library Collections REST API views
+"""
+
+from __future__ import annotations
+
+from openedx_learning.api.authoring_models import Collection
```

Comment:
Could you use the `openedx_learning.api.authoring` API instead of creating Collection objects directly?